website/docs/Custom-Rules.md

```diff
@@ -45,7 +45,7 @@ rule "MyCustomRule"
 
   propertydefinition {
     name = "StripDebugInfo",
-    ind = "boole
+    ind = "boolean",
```

Comment:
I believe this should also be `kind` rather than `ind`?

Reply:
I think so as well.  Not 100% sure what happened to this file, but it appears that it got messed up.